lib_xud/src/core/XUD_IoLoop.S

```diff
@@ -200,12 +199,15 @@ ConfigRxA:                                      // Configure a event on RXA goin
     setv        res[RXA], r11
     eeu         res[RXA]
 
-ConfigValidToken:
 #ifdef __XS2A__
-    ldw         r10, sp[STACK_VTOK_PORT]        // ValidToken is used for timing timeout period when expecting an ack after tx
-#else
-    ldw         r10, dp[rx_rdy]                 // TODO use from stack
+ClearValidToken:                               // Clear any VTok port condition
+    ldw         r10, sp[STACK_VTOK_PORT]
+    setc        res[r10], XS1_SETC_COND_NONE
 #endif
+
+ConfigTxReadyInVector:                          // Configure event on TxReadyIn for timing timeout when waiting ack after tx, can use any usb clocked port here
```

Comment:
Following my above comment, this was my naming choice, I know, but I guess "Vector" should be removed from this label if I was being super picky.